bower.json

```diff
@@ -29,7 +29,7 @@
   ],
   "dependencies": {
     "d3": "3.3.9",
-    "ember": "~1.1.0",
+    "ember": "~1.10.0",
```

Comment:
I need to look into whether or not these changes will require Ember 1.10, or just support it. I'm very happy with making things work on Ember 1.10 but I want to ensure we don't break backwards compatibility for ember-charts users still on earlier versions of Ember.